### PR TITLE
{devel,vis}[intel/2016b] Yade v2016.06a, VTK v6.3.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2034.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/bb6d0baee6b61b9e6cee0292f0598f16 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/69a0be56636dc3b91cefbf2503beae12 for a full test report.

File: easybuild/easyconfigs/y/Yade/Yade-2016.06a-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['https://launchpad.net/yade/trunk/yade-1.00.0/+download/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['Yade-%(version)s_include-signal.patch']
+
+dependencies = [
+    ('Boost', '1.61.0', versionsuffix),
+    ('CGAL', '4.8.1', versionsuffix),
+    ('Eigen', '3.2.9'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/y/Yade/Yade-2016.06a-intel-2016b-Python-2.7.12.eb]: build dep?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9536dd7409e640b3aca19ec545d56f1c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2011.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/74d081cb951eae4cf83b207dd9189909 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**